usersync/usersyncers/syncer.go

```diff
@@ -26,6 +26,16 @@ func NewSyncerMap(cfg *config.Configuration) map[openrtb_ext.BidderName]usersync
 	}
 }
 
+func ParseIdsGDPR(syncers map[openrtb_ext.BidderName]usersync.Usersyncer) map[openrtb_ext.BidderName]uint16 {
+	gdprAwareSyncers := make(map[openrtb_ext.BidderName]uint16, len(syncers))
```

Comment:
Not entirely happy with this name, as to me it implies the values should be  syncers, not GDPR IDs. perhaps rename to `gdprAwareSyncerIDs`?

Reply:
Hah, actually referred to the first variable in the function rather than the function name itself, but this is good too. :)